### DISCUSSION TITLE
Should ansible be a dependency?

### DISCUSSION THREAD

**Discussion by darrenmeehan** [ansible/ansible-lint#1847] [Q&A]
Hi there, 

I was wondering should `ansible-lint` have `ansible` as a dependency? It cannot function without it, so it makes sense to list it is a dependency. Is there any reason it is not?

The particular use-case I have is that I want to use `pipx` to install ansible-lint into its own virtual environment.
This currently doesn't work. I've not opened this as an issue in either pipx or ansible-lint as I'd like to investigate this a bit more first.

The following output shows `ansible-lint` unable to find `ansible`.

```
$ ansible-lint
ERROR    No module named 'ansible'
FATAL: ansible-lint requires a version of Ansible package >= 2.9, but none was found. Please install a compatible version using the same python interpreter. See https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html#installing-ansible-with-pip
```

The following is `pipx` installing `ansible-lint`

```
pipx install --include-deps ansible-lint --force
Installing to existing venv 'ansible-lint'
⚠️  Note: cmark was already on your PATH at /usr/local/bin/cmark
⚠️  Note: pygmentize was already on your PATH at /usr/local/bin/pygmentize
  installed package ansible-lint 5.3.2, installed using Python 3.10.1
  These apps are now globally available
    - ansible-lint
    - cmark
    - pygmentize
done! ✨ 🌟 ✨
```

**Comment by nroach44** [comment]
Just thought I'd drop this in here, if you install `ansible-lint`, you can have your ansible tracked in the same venv with:

`pipx inject ansible-lint ansible-core --include-apps`

(if you don't already have an ansible / ansible-core install)
